Find a functionally equivalent code:android.media.AudioService.sendMasterVolumeUpdate:COMMENT
Method Modifier: private     
Comment:// UI update and Broadcast Intent

@@ -1,5 +1,5 @@
 {
-    mVolumePanel.postMasterVolumeChanged(flags);
+    mVolumeController.postMasterVolumeChanged(flags);
     Intent intent = new Intent(AudioManager.MASTER_VOLUME_CHANGED_ACTION);
     intent.putExtra(AudioManager.EXTRA_PREV_MASTER_VOLUME_VALUE, oldVolume);
     intent.putExtra(AudioManager.EXTRA_MASTER_VOLUME_VALUE, newVolume);

